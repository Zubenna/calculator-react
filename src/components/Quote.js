import React from 'react';

const Quote = () => (
  <div>
    <h1>Math Magicians</h1>
    <p>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding.
      â€“William Paul Thurston
    </p>
  </div>
);

export default Quote;
