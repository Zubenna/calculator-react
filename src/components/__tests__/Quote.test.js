import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Quote', () => {
  it('Tests the rendering of components', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});