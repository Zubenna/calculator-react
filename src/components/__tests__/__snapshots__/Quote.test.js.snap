// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Tests the rendering of components 1`] = `
<div
  className="quote"
>
  <h1
    className="quote-title"
  >
    Math Magicians
  </h1>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
